ksqldb-engine/src/test/java/io/confluent/ksql/execution/ExpressionEvaluatorParityTest.java

```diff
@@ -174,10 +174,24 @@ public void shouldDoArithmetic() throws Exception {
   @Test
   public void shouldDoArithmetic_nulls() throws Exception {
     ordersRow = GenericRow.genericRow(null, null, null, null, null, null, null, null, null);
-    assertOrdersError("1 + null",  compileTime("Unexpected error generating code for Test"),
-        compileTime("Unexpected error generating code for expression: (1 + null)"));
-    assertOrdersError("'a' + null",  compileTime("Unexpected error generating code for Test"),
-        compileTime("Unexpected error generating code for expression: ('a' + null)"));
+    assertOrdersError("1 + null",  compileTime("Error processing expression: (1 + null)."),
```

Comment:
I think most of these should effectively have the same error message for each of the interpreter and compiler, since it's the same thing throwing the error. So you can use the same message, to make clear they are the same error.